general/compatibility.md

```diff
@@ -0,0 +1,31 @@
+---
+description: This is a resume of compatibility between the components of the Pantos ecosystem.
+---
+
+# Compatibility between versions
+
+## The Pantos ecosystem
+
+The Pantos ecosystem is composed of:
+- [Pantos protocol](https://github.com/pantos-io/ethereum-contracts)
+  
+  A collection of smart contracts implementing the Pantos protocol.
+
+- [Service Node](https://github.com/pantos-io/servicenode)
+  
+  A baseline implementation of a node interacting with the Pantos protocol and the multiple supported blockchains.
+
+- [Validator Node](https://github.com/pantos-io/validatornode)
+
+  A node responsible for validating cross-chain transfers on behalf of the users.
+
+The Service Node interacts with the Pantos protocol to initiate cross-chain transfers. In order to do so, the Service Node needs to be compatible with the current version of the Pantos protocol. 
```

Comment:
The off-chain components (validator nodes, service nodes, clients) also implement part of the Pantos protocol. I would clearly distinguish between the protocol version and the contracts version. I think we should define protocol version 1.0 to be the protocol at mainnet launch and stick with a 0.x protocol version until then.

Reply:
Addendum: Not every contracts modification/release necessarily means a protocol modification.